grass7/imagery/i.landsat/i.landsat.download/i.landsat.download.py

```diff
@@ -238,7 +251,7 @@ def main():
                 print(
                     scene["entityId"],
                     scene["displayId"],
-                    scene["acquisitionDate"],
```

Comment:
I think there are several ways to go, depending on the pending `landsatxplore` release. In any case, for the nested structure of the metadata, you could use for example:
`scene['temporalCoverage']['startDate']` --> `scene['temporalCoverage']` is a dictionary itself with the two keys `startDate` and `endDate`
or 
`scene['start_time']`, 
both returning strings that might need further handling if you need them to be datetime objects (?)

Reply:
Sorry, I realised I misunderstood the issue. Will look into the sorted()-problem

Reply:
> Sorry, I realised I misunderstood the issue. Will look into the sorted()-problem

no problem... Let's see how the `acquisition_date` problem is fixed in landsatxplore. If that is harmonized properly and delivers a properly formatted date, then we can use it directly.

Reply:
Both the `date` and the `cloudCover` that we can get from the metadata are string objects. This means they have to be converted to float/datetime objects before they can be sorted in a meaningful way. Now the date might already be a `datetime` object in the new `landsatxplore` release. However for now I would suggest the following in lines 233+ (sorry I can't directly comment there, because these lines werent changed by this PR):
```
        sort_vars = options["sort"].split(",")

        reverse = False
        if options["order"] == "desc":
            reverse = True

        scenes_extracted = []
        for idx,scene in enumerate(scenes):
            scene_dict = {
                'idx': idx,
                'cloudCover': float(scene['cloudCover']),
                'acquisitionDate': datetime.strptime(scene['temporalCoverage']['startDate'], '%Y-%m-%d %H:%M:%S')
            }
            scenes_extracted.append(scene_dict)
        scenes_extracted_sorted = sorted(scenes_extracted, key=lambda i: (i[sort_vars[0]], i[sort_vars[1]]), reverse=reverse)
        sorted_idcs = [scene['idx'] for scene in scenes_extracted_sorted]
        sorted_scenes = [scenes[i] for i in sorted_idcs]

```
I can't think of a way to convert the values 'on the fly' (but probably there is :) ) so this way would go by creating an auxiliary list of dictionaries.
Later when you print, you can just directly print `scene["temporalCoverage"]['startDate']`